examples/virtualenv/projects/MLproject

```diff
@@ -0,0 +1,5 @@
+name: virtualenv-example
+python_env: python_env.yaml
```

Comment:
Do we want to support running a project containing `conda_env` with `--env-manager=virtualenv`?

```
name: virtualenv-example
conda_env: conda.yaml
```

Reply:
Yep, that would be great!